### ISSUE TITLE
A11y_PowerShell Core[Benchmark]_PowerShell Editor Services_Installing Powershell in Vscode_Screen Reader: VoiceOver is not announcing about "Rating" information about the PowerShell.

### ISSUE THREAD

**Issue by Gayathri1501** [microsoft/vscode#148888]
GitHub Tags:
#A11yMAS; #A11ySev1; #A11yTCS; #DesktopApp; #BM-PowerShellCore-MAC-Jan21; #PowerShell Core; #MAS1.3.1; #MAC; #AINotInScope; #Info and Relationships; #Voiceover; #Element: Button; #ScreenReader; #Benchmark.

### Environment Details:​
Application Name: Visual Studio Code Version: 1.66.2(user setup)
PowerShell Verion: v2022.5.0
MAC Version: macOS Monterey 12.2.1

### Repro Steps:

1. Start Voiceover with using shortcut key Command+F5.
2. Download VS Code Space Insiders 
3. Open VS Code Space insiders 
4. Tab till Extension in the left navigation and press enter
5. Tab till Search Extensions in Marketplace and type PowerShell and press enter
6. Tab till Rating information check whether Voiceover is reading the rating information such as 4 stars rated for PowerShell or not

### Actual Result:​
Voiceover is not announcing about "rating" information about the PowerShell.

### Expected Result:​
Voiceover should be announcing about "Rating" information about the PowerShell for the users to understand how much PowerShell is rated
### Attachment:
![9128131_A11y_PowerShell Core Benchmark _PowerShell Editor Services_Installing PowerShell in Vscode_Screen Reader](https://user-images.githubusercontent.com/99962579/167099843-59842abe-5776-4763-b8fb-6f506ce31556.gif)

**Comment by isidorn** [comment]
Issue is fixed. Adding #Closed; tag.
